utils/issue-format-bot/lib/issueedit.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 AJ Jordan
+//
+// This file is part of the HTTPS Everywhere issue management bot.
+//
+// The HTTPS Everywhere issue management bot is free software: you can
+// redistribute it and/or modify it under the terms of the GNU Affero
+// General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// The HTTPS Everywhere issue management bot is distributed in the
+// hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+// the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+// PURPOSE.  See the GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with the HTTPS Everywhere issue management bot.  If
+// not, see <http://www.gnu.org/licenses/>.
+
+'use strict';
+
+const parse = require('./parse'),
+      validate = require('./validate'),
+      labeler = require('./labeler'),
+      _ = require('lodash');
+
+// TODO make this share more with newissue.js?
+
+module.exports = function(robot, alexa) {
```

Comment:
That's true but it seems more elegant this way? In particular it matches Probot documentation more closely, and I think `robot.log` makes the fact that you're logging to the robot more obvious. Also if you passed in `robot.log` instead of `robot` you'd have to add a new argument every time you wanted to use something new from `robot` (and then adjust the test suite to match).